docs/source/science/ESA_CCI/ESA_CCI_V4.ipynb

```diff
@@ -17,52 +17,39 @@
     "tags": []
```

Comment:
<p>Question. So, we chose to plot the time series for data points &lt; 10, after observing that the average hovers around 200. </p><p> </p><p>To me, it means we're saying "let's look at how the amount of biomass evolved in places where there wasn't much biomass in the first place"</p><p> </p><p>Am I right ? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/335/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
<p>I would say the idea here is to plot the mean-time series from the filtered data based on the issue, irrespective of the average values. So, trying to answer - "show me the mean time-series for the AGB estimate &gt; (or &lt;) 10 for the selected region"?</p><p> </p><p>Does it makes sense?</p>

Reply:
<blockquote>I would say the idea here is to plot the mean-time series from the filtered data based on the issue, irrespective of the average values</blockquote><p> </p><p>So, you are saying that we're just sticking to the instructions provided by the user in the initial ticket. That makes sense to me. </p><p> </p><p>With my comment, I was trying to go a little further than that, by trying to find a rationale for WHY we chose 10 and not any other number, say, 50, 100, or 150. We're not justifying why we selected 10. Although providing such a justification would make for a more intuitive and understandable example, let's just move on, and merge as is. </p>